### PR TITLE
Translate minio-dotnet API docs to Chinese.

### PR DESCRIPTION
**Pull request by dingjs**
Translate minio-dotnet API docs to Chinese.

### CODE REVIEW

File: Docs/zh_CN/API.md [hunk 1]
```diff
@@ -0,0 +1,1271 @@
...
+| `accessKey`   | _string_   |accessKey类似于用户ID，用于唯一标识你的账户。可选，为空代表匿名访问。 |
+|`secretKey`  |  _string_   | secretKey是你账户的密码。可选，为空代表匿名访问。 |
+|`region`  |  _string_   | 对象存储的region。可选。|
+
+__安全访问__
+
+|  |
+|---|
+|`针对client对象链式调用.WithSSL()，可以启用https。 `   |
+
```
Comment (poornas) [inline Docs/zh_CN/API.md]: @dingjs,ran this doc through Google translate - it is not able to translate line 62, you might want to double check this.

Comment (dingjs) [reply Docs/zh_CN/API.md]: Thanks,I checked this line and modified it,remove `“针对”` would be better.

File: README_zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                Console.WriteLine("File Upload Error: {0}", e.Message);
+            }
+        }
+    }
+}
+```
+
+## 运行Minio Client示例
+####  Windows
+* clone这个repository，并在Visual Studio 2017中找开Minio.Sln。
```
Comment (yihuaf) [inline README_zh_CN.md]: 找开->打开

File: README_zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+* 在Minio.Examples/Program.cs中输入你的认证信息、存储桶名称、对象名称等。
+  在Program.cs中取消注释以下示例测试用例来运行示例。
```
Comment (yihuaf) [inline README_zh_CN.md]: 取消注释以下类似的测试用例来运行示例

File: README_zh_CN.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+```
+##### 运行Minio.Examples 
+* clone这个项目。
+
+```
+$ git clone https://github.com/minio/minio-dotnet && cd minio-dotnet 
+```
+
+* 在Minio.Examples/Program.cs中输入你的认证信息、存储桶名称、对象名称等。
+  在Program.cs中取消注释以下示例测试用例来运行示例。
```
Comment (yihuaf) [inline README_zh_CN.md]: ditto

**Comment by dingjs** [comment]
@yihuaf  ping.

**Pull request merged**